docs/resources/equinix_fabric_connection.md

```diff
@@ -1,7 +1,7 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
 page_title: "equinix_fabric_connection Resource - terraform-provider-equinix"
-subcategory: "Fabric"
+subcategory: ""
```

Comment:
This must be "Fabric" to appear listed in the Fabric sub-section of the registry, tf-plugins-docs does not take it into account